src/App.css

```diff
@@ -47,11 +69,33 @@ tr {
   padding-top: 15px;
 }
 
+.touristic-cards-container {
+  margin: 50px 50px;
+  display: flex;
+  flex-direction: row;
+  justify-content: space-around;
+}
+
 .footer {
+  bottom: 0;
+  width: 100%;
+  position: fixed;
```

Comment:
This is nice 🙂 

I'd recommend not over-using this though - some people find floating headers/footers annoying.